qcut/docs/issues/video_play/consolev3.md

```
@@ -0,0 +1,246 @@
+blob-manager.ts:193 [BlobManager] 📉 Released: blob:app://./f41436d1-c371-4a9d-8095-2ac30ed96641
+blob-manager.ts:194   📍 Created by: media-store-display
+blob-manager.ts:195   🔄 Released by: VideoPlayer-unmount
+blob-manager.ts:196   📊 Remaining refs: 1
+blob-manager.ts:71 [BlobManager] ♻️ Reusing URL (instance match): ecbb2587-d623-2eb2-109f-e8306a658615
+blob-manager.ts:74   📍 Original source: media-store-display
+blob-manager.ts:75   🔄 Requested by: VideoPlayer
+blob-manager.ts:76   📊 Ref count: 2
+video-player.tsx:184 [VideoPlayer] Using blob URL for ecbb2587-d623-2eb2-109f-e8306a658615: blob:app://./f41436d1-c371-4a9d-8095-2ac30ed96641
+video-player.tsx:260 [VideoPlayer] ✅ Video loaded: ecbb2587-d623-2eb2-109f-e8306a658615
+video-player.tsx:279 [VideoPlayer] ▶️ Video ready to play: ecbb2587-d623-2eb2-109f-e8306a658615
+canvas-utils.ts:21 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)